src/librustc/middle/stability.rs

```diff
@@ -432,6 +432,18 @@ struct Checker<'a, 'tcx: 'a> {
 }
 
 impl<'a, 'gcx, 'tcx> TyCtxt<'a, 'gcx, 'tcx> {
+    fn skip_stability_check_due_to_privacy(self, def_id: DefId) -> bool {
```

Comment:
I'm not sure I follow.

The check you are referring to is `fn check_missing_stability`, right?

Is there an example you can show me of input code that is currently emitting a "missing annotation" error that you believe should be skipped via this check?